src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int INDEX_OF_NUMBER = 1;
+    private static final int SUPPLY_POSITION = 0;
+    private static final int BUY_POSITION = 1;
+    private static final int RESULT_POSITION = 2;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        WorkWithFile worker = new WorkWithFile();
+        worker.writeData(worker.createReport(worker.readFromFile(fromFileName)), toFileName);
+    }
+
+    public String readFromFile(String fromFileName) {
+        StringBuilder builder = new StringBuilder();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read a file", e);
+        }
+        return builder.toString();
+    }
+
+    public int[] createReport(String string) {
```

Comment:
```suggestion
    public String createReport(String string) {
```
Return a line that is ready to be written to the file, so you don't have to do extra logic in the writeData method